import_export/tmp_storages.py

```diff
@@ -69,23 +70,50 @@ def remove(self):
 
 class MediaStorage(BaseStorage):
     MEDIA_FOLDER = "django-import-export"
+    _storage = None
 
     def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        if django.VERSION[:2] > (4, 2):
```

Comment:
Small factoring comment: I'd personally lean toward moving this into sub functions e.g. _configure_storage_django_42 and _configure_storage_django_legacy or something. If we had more than 2 versions maybe a factory _configure_storage method that picks which version function to call.

Reply:
Good call.  I've made this change.